security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -572,10 +572,10 @@ func (sc *SecretCache) rotate(updateRootFlag bool) {
 
 		if sc.configOptions.CredFetcher != nil {
 			// Refresh token through credential fetcher.
-			cacheLog.Infof("%s getting a new token through credential fetcher", logPrefix)
+			cacheLog.Debugf("%s getting a new token through credential fetcher", logPrefix)
 			t, err := sc.configOptions.CredFetcher.GetPlatformCredential()
 			if err != nil {
-				cacheLog.Errorf("credential fetcher failed to get token: %v", err)
+				cacheLog.Warnf("credential fetcher failed to get a new token, continue using the original token: %v", err)
```

Comment:
wondering why this line is not following logPrefix?

Reply:
Good catch. I added logPrefix. Thanks.

Reply:
Nice catch @incfly